tests/load/script_archive_unused/generate_raw_transaction_fixed.py

```
@@ -0,0 +1,215 @@
+import rlp
+from pathlib import Path
+
+
+def load_contract_code():
+    """Load the WizardOfCoin contract code from the examples directory."""
+    contract_path = Path(__file__).parent.parent.parent / "examples" / "contracts" / "wizard_of_coin.py"
+    with open(contract_path, 'r') as f:
+        return f.read()
+
+
+def encode_genlayer_calldata(constructor_args):
```

Ruff (0.12.2)

Unused function argument: `constructor_args`

(ARG001)